### PR TITLE
Report an error for each additional property.

### PR DESCRIPTION
**Pull request by sergey-shandar**

### CODE REVIEW

File: src/JsonValidation.js [hunk 1]
```diff
@@ -208,7 +208,9 @@ var JsonValidators = {
                     }
                 }
                 if (s.length > 0) {
-                    report.addError("OBJECT_ADDITIONAL_PROPERTIES", [s], null, schema);
+                    for (const i of s) {
+                        report.addError("OBJECT_ADDITIONAL_PROPERTIES", [i], null, schema);
```
Comment (vladbarosan) [inline src/JsonValidation.js]: i think missing required properties also reports errrors in an aggregated way

Comment (sergey-shandar) [reply src/JsonValidation.js]: @vladbarosan do we have any case where we would like to suppress missing required property?

**Comment by zaggino** [comment]
@sergey-shandar can you please add a test-case to this?

**Comment by zaggino** [comment]
published `z-schema@3.25.0`

**Pull request merged**